src/_components/routes.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+import Home from '../components/Home/Home';
+import ResetPassword from '../components/ForgotPassword/ResetPassword';
+import ForgotPassword from '../components/ForgotPassword/ForgotPassword';
+import ForgotPasswordMessage from '../components/ForgotPassword/ForgotPasswordMessage';
+import SignUp from '../components/Auth/SignUp';
+// import Auth from '../components/Auth';
+
+const Routes = () => (
+  <div>
+    <Route exact path="/" component={Home} />
+    <Route exact path="/" component={ForgotPassword} />
+    <Route exact path="/signup" component={SignUp} />
+    {/* <Route exact path="/auth" component={Auth} /> */}
```

Comment:
commented code

Reply:
I am going to remove it in the next commit!